### PR TITLE
Debug commands

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

About `symfony/var-dumper`: it would be great to be able to go to the file over click on the class/method/file in the output thanks to https://github.com/symfony/symfony/pull/31446/files

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -93,7 +95,8 @@
         "sort-packages": true,
         "allow-plugins": {
             "infection/extension-installer": true,
-            "composer/package-versions-deprecated": true
+            "composer/package-versions-deprecated": true,
+            "yiisoft/config": false
```
Comment (rustamwin) [inline composer.json:99]: ?

Comment (xepozz) [reply composer.json:99]: It was automatically after config installation

File: src/Command/DebugContainerCommand.php [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        $config = $this->container->get(ConfigInterface::class);
+
+        $io = new SymfonyStyle($input, $output);
+
+        if ($input->hasArgument('id') && !empty($ids = $input->getArgument('id'))) {
+            $build = $this->getConfigBuild($config);
+            foreach ($ids as $id) {
+                $definition = null;
+                foreach ($build as $definitions) {
+                    //var_dump(array_keys($definitions));
```
Comment (samdark) [inline src/Command/DebugContainerCommand.php:57]:
```suggestion
```

File: src/Command/DebugContainerCommand.php [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+                    if (array_key_exists($id, $definitions)) {
+                        $definition = $definitions[$id];
+                    }
+                }
+                if ($definition === null) {
+                    $io->error(
+                        sprintf(
+                            'Service "%s" not found',
```
Comment (samdark) [inline src/Command/DebugContainerCommand.php:65]:
```suggestion
                            'Service "%s" not found.',
```

File: src/Command/DebugEventsCommand.php [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        $data = [];
+        if ($config->has('events')) {
+            $data = array_merge($data, $config->get('events'));
+        }
+        if ($config->has('events-console')) {
+            $data = array_merge($data, $config->get('events-console'));
+        }
+        //if ($config->has('events-web')) {
+        //    $data = array_merge($data, $config->get('events-web'));
+        //}
```
Comment (samdark) [inline src/Command/DebugEventsCommand.php:87]:
```suggestion
```

**Comment by samdark** [comment]
👍

**Pull request merged**